Deploy/Systemd/ss-susi-linux.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Starting SUSI Linux 
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+Type=idle
+ExecStart=/usr/bin/python3 /home/pi/SUSI.AI/susi_linux/access_point/server/server.py
```

Comment:
same here also

Reply:
Why the command here, `access_point/server/server.py`, is the same as _ss-python-flask.service_?

Reply:
@hongquan Here, we are using `RemainAfterExit` because we want the server to be always ON